release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -86,6 +86,68 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli_{context}"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-mirror-v2_{context}"]
+==== oc-mirror plugin v2 (Technology Preview)
+
+The oc-mirror plugin v2 for {product-title} includes new features and functionalities that improve the mirroring process for Operator images and other {product-title} content.
+
+The following are the key enhancements and features in oc-mirror plugin v2:
+
+* *Automatic generation of IDMS and ITMS objects*:
+
+oc-mirror plugin v2 automatically generates a comprehensive list of `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) objects after each run. These objects replace the `ImageContentSourcePolicy` (ICSP) used in oc-mirror plugin v1. This enhancement eliminates the need for manual merging and cleanup of operator images and ensures all necessary images are included.
+
+* *CatalogSource objects*: 
+
```

Comment:
Would adding a plus befroe each description help indent the text with the description-list heading?

Reply:
Done